plugins/SegmentEditor/SegmentSelectorControl.php

```diff
@@ -68,7 +68,7 @@ public function __construct($idSite = false)
         $this->nameOfCurrentSegment = '';
         $this->isSegmentNotAppliedBecauseBrowserArchivingIsDisabled = 0;
 
-        $this->availableSegments = Request::processRequest("SegmentEditor.getAll", ['idSite' => $this->idSite], $defaultRequest = []);
+        $this->availableSegments = SegmentEditor::getAllSegmentsForSite($this->idSite);
```

Comment:
This change reorders the segments in the frontend matching the [UI test failure](https://builds-artifacts.matomo.org/matomo-org/matomo/dev-17463/6944483032/SegmentSelectorEditorTest_updated.png).

Is that an intended/good change or should the ordering in `getAllSegmentsForSite` be adjusted?

Reply:
I've just checked the places again where that method actually was used before and actually nowhere did the order matter. So I'll simply remove the reordering in `SegmentEditor::getAllSegmentsForSite`